src/cli/subcommands/sync_cmd.rs

```diff
@@ -123,11 +123,12 @@ impl SyncCommands {
 
                 let height_diff = base_height - target_height;
 
-                // If the sync state is not in the Complete stage and both base and target tipsets are empty,
+                // If the sync state is not in the Complete stage
+                // and both base and target cid's are empty,
                 // the node might be downloading the snapshot.
                 if state.stage() != SyncStage::Complete
-                    && base_cids.is_empty()
-                    && target_cids.is_empty()
+                    && base_cids.eq("[]")
```

Comment:
Let's make `"[]"` a constant

Reply:
Instead of returning "[]" can we do something like this:

` fn format_tipset_cids(cids: &str) -> &str {
       if cids == "[]" { "None" } else { cids }
   }`
   
 this way it will be more readable otherwise it's little confusing?

Reply:
Basically return an empty string and format it while printing.

Reply:
Sounds good

Reply:
Done